src/ui/public/styles/pagination.less

```diff
@@ -42,6 +42,7 @@ paginate {
             text-decoration: none !important;
             font-weight: bold;
             color: @paginate-page-link-active-color;
+            cursor: default;
```

Comment:
Typically these kinds of rules reset another rule that had set `cursor: pointer`